buildimage.sh

```diff
@@ -2,11 +2,13 @@
 mvn clean install
 rm -rf docker/lib/ext
 mkdir -p docker/lib/ext
+cp appengine-jetty-managed-runtime/target/jetty-distribution.tar.gz docker/
```

Comment:
Would it be better to untar outside of the Docker build step? Since Docker is using copy on write, the image would otherwise contain the tar.gz and it exploded content...
The goal is to try to always minimize the image size.